datumaro/plugins/data_formats/datumaro/exporter.py

```diff
@@ -92,10 +92,9 @@ def context_save_media(
 
             if context.save_media:
                 # Temporarily update image path and save it.
-                image._path = osp.join(
-                    context.images_dir, item.subset, context.make_image_filename(item)
-                )
-                context.save_image(item, encryption=encryption, path=image.path)
+                fname = context.make_image_filename(item)
```

Comment:
I have one question, I think set `fname` through `make_image_fillename` could cover in `save_image` by returning `fname` result. Is there any reason to call it separately?

Reply:
This is because there is an exceptional case which requires `fname` to be pre-defined outside of `make_image_filename()`:
https://github.com/openvinotoolkit/datumaro/blob/e416f3e39361d77a96127057c290a157ff67c452/datumaro/plugins/data_formats/datumaro/exporter.py#L112-L123